lib/resources/r4/related_person_patch.yaml

```diff
@@ -131,6 +142,46 @@ operations:
         <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
       </ul>
 
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` of the relationship at the given <code>{index}</code>.
+    example: |
+      {
+        "op": "replace",
+        "path": "/relationship/0/extension",
+        "value": [
+          {
+            "valuePeriod": {
+              "start": "2016-12-30T22:26:08.000Z",
+              "end": "2036-12-30T22:26:08.000Z"
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
+          },
+          {
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/40",
+                  "code": "155",
+                }
+              ]
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Requires a test operation to be provided for the `Relationship.id` of the relationship whose `period` and `relation` is intended to be replaced.</li>
```

Comment:
Will 'test operation' need to be linked like the other patch operations?

Reply:
+1. We should change it to:
`Requires a <a href="#test-relationship-id">test operation</a> .....`

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/662/commits/d55126717a21ef75564d0c386dd49a8801793c8f#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR176

Reply:
+1 to changes.